### ISSUE TITLE
[Bug]: Need Help "View Documentation" button generates 404 Page Not Found

### ISSUE THREAD

**Issue by kiyote33** [code4recovery/12-step-meeting-list#916]
### Contact Details

waynep@code4recovery.org

### Website With Issue

https://code4recovery.org/docs/12-step-meeting-list

### What happened?

On the 12 Step Meeting List Import & Settings page, tab on Settings. Click on "View Documentation". Expect to see help for TSML.
Instead get the old "Oops! That Page Can't Be Found" message displaying on the [Code For Recovery](https://code4recovery.org) site. If I go to the site and click on the embedded page links it takes me to https://wordpress.org/plugins/12-step-meeting-list/#my%20meeting%20type%20isn%27t%20listed%21 which works, but that is an external link to the site. 

We either need a redirect on the Code For Recovery or modify TSML's Help "Doc" button to use the [Word Press link](https://wordpress.org/plugins/12-step-meeting-list/#my%20meeting%20type%20isn%27t%20listed%21).

### TSML Version

Latest (default)

### Wordpress Version

Latest (default)

### What browsers are you seeing the problem on?

Microsoft Edge

**Comment by tim-rohrer** [comment]
I believe we can move this to our website repo issues. 

We have agreed not to use the Wordpress plugin site because we want to keep support consolidated.

Is that okay?

**Comment by joshreisner** [comment]
It would be great to agree on a single place. Earlier we had discussed consolidating on the WordPress site because it’s up to date. The Github wiki page hasnt been updated in a long time

**Comment by tim-rohrer** [comment]
Yes! We need a push to update the docs. Hopefully we’re all in agreement still that our Wordpress site as a portal with GitHub repos as “source” is the right approach. 

Perhaps I’m misunderstanding, but I don’t believe we should be linking to the Wordpress Plugin site at Wordpress.org.

**Comment by joshreisner** [comment]
the tsml team should pick a place for  documentation and delete the other places we maintain copies of it, to eliminate confusion. the wordpress plugin page on wordpress.org, updated via readme.txt, is currently the most recently updated.

**Comment by joshreisner** [comment]
i've created a poll on slack

**Comment by joshreisner** [comment]
- [x] decide to use https://code4recovery.org/docs/12-step-meeting-list as our documentation link globally, leave that page published (by reviving this URL, the button in the plugin is fixed now 🎉 )
- [x] remove https://code4recovery.org/faqs (not linked from anywhere)
- [x] turn off github wiki (checked for links on [meetingguide.helpdocs.io](https://meetingguide.helpdocs.io/), none found)
- [x] temporarily edit https://code4recovery.org/docs/12-step-meeting-list with a link saying "while this page is under construction, use WordPress FAQ page"
- [x] apply Lois's update to WordPress FAQ (in `main`, will go live in next release)
- [ ] research / discover / implement way to reproduce WP FAQ on https://code4recovery.org/docs/12-step-meeting-list, while maintaining a single-file always-in-sync source of truth (assigned to member Andy B)
